notebooks/advanced_RAG.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    retrieved_docs = KNOWLEDGE_INDEX.similarity_search(query=user_query, k=5)
</pre><p>Any advice on choosing the <code>k</code> number?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Just added a section on that, and the relation with the chunk size!